### PR TITLE
Implemented strftime in PayloadEncoder and ESJsonEncoder (#1508).

### PR DESCRIPTION
**Pull request by jotes**
@rafrombrc @trink could you review? Thanks in advance.

### CODE REVIEW

**Comment by rafrombrc** [comment]
Overall this looks good, just need to add ESLogstashV0Encoder to the list of changes in the changelog, and update the docs for PayloadEncoder and ESLogstashV0Encoder to reflect use of the strftime format.

**Comment by jotes** [comment]
Okay, will do.

On Tue, May 26, 2015 at 10:42 PM, Rob Miller <notifications@github.com>
wrote:

> Overall this looks good, just need to add ESLogstashV0Encoder to the list
> of changes in the changelog, and update the docs for PayloadEncoder and
> ESLogstashV0Encoder to reflect use of the strftime format.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla-services/heka/pull/1550#issuecomment-105659042>
> .
>



-- 
jarek@reijutsu:~$ fortune
You have Egyptian flu: you're going to be a mummy.
jarek@reijutsu:~$ fortune
You now have Asian Flu.

File: CHANGES.txt [hunk 1]
```diff
@@ -8,7 +8,7 @@ Backwards Incompatibilities
 
 * LogOutput will write data to stdout instead of stderr (#1515).
 
-* Using stftime literals for filenames during rotation in FileOutput plugin (#1469).
+* Implemented stftime format codes in: filenames in FileOutput plugin, ESJSONEncoder, Payload encoder (#1469, #1508)
```
Comment (rafrombrc) [inline CHANGES.txt]: Add mention of ESLogstashV0Encoder.

File: docs/source/config/encoders/esjson.rst [hunk 2]
```diff
@@ -39,8 +39,8 @@ Config:
     all dynamically specified message fields. Defaults to including all of the
     supported message fields.
 - timestamp (string):
-    Format to use for timestamps in generated ES documents. Defaults to
-    "2006-01-02T15:04:05.000Z".
+    Format to use for timestamps in generated ES documents. Allows to use
+    strftime format codes. Defaults to"%Y-%m-%dT%H:%M:%S".
```
Comment (rafrombrc) [inline docs/source/config/encoders/esjson.rst]: Typo, missing a space after the "to".

File: docs/source/config/encoders/eslogstashv0.rst [hunk 3]
```diff
@@ -49,8 +49,8 @@ Config:
     supported message fields. The "Payload" field is sent to ElasticSearch as
     "@message".
 - timestamp (string):
-    Format to use for timestamps in generated ES documents. Defaults to
-    "2006-01-02T15:04:05.000Z".
+    Format to use for timestamps in generated ES documents. Allows to use
+    strftime format codes. Defaults to"%Y-%m-%dT%H:%M:%S".
```
Comment (rafrombrc) [inline docs/source/config/encoders/eslogstashv0.rst]: Typo, missing a space after the "to".

File: CHANGES.txt (hunk 1 above)
Comment (jotes) [inline CHANGES.txt]: I was sure that i've puhed updated version of docs. I'll fix this now.

**Pull request merged**